### ISSUE TITLE
Allow removing the LRC site from the service

### ISSUE THREAD

**Issue by MrCreosote** [kbase/cdm-task-service#600]
The Lawrencium / LRC site is currently not available from JAWS. I attempted to do minimal surgery to remove it from the sites endpoint (where it should be marked as inactive) but it turned into a mess. The issue is that it's baked into an enum and a number of spots expect it to be present in the enum, including mongo. It's also coupled with the NERSC runner reasonably tightly.

Need to rework things so that the list of sites isn't an enum (but ideally Pydantic / FastAPI can still validate it) most likely. Need some thought.

**Comment by MrCreosote** [comment]
Startup still failed after this
```diff
$ git diff
diff --git a/cdmtaskservice/app_state.py b/cdmtaskservice/app_state.py
index 602a2c8..838b578 100644
--- a/cdmtaskservice/app_state.py
+++ b/cdmtaskservice/app_state.py
@@ -335,7 +335,7 @@ async def _register_nersc_job_flows(
     )
     dest.register("JAWS flow provider", jaws_job_flows.close())
     flowman.register_flow(NERSCJAWSRunner.CLUSTER, jaws_job_flows.get_nersc_job_flow)
-    flowman.register_flow(LawrenciumJAWSRunner.CLUSTER, jaws_job_flows.get_lrc_job_flow)
+    # flowman.register_flow(LawrenciumJAWSRunner.CLUSTER, jaws_job_flows.get_lrc_job_flow)
     return jaws_job_flows
 
 
diff --git a/cdmtaskservice/jaws/sitemapper.py b/cdmtaskservice/jaws/sitemapper.py
index 5e442cc..98925ec 100644
--- a/cdmtaskservice/jaws/sitemapper.py
+++ b/cdmtaskservice/jaws/sitemapper.py
@@ -11,7 +11,7 @@ _CLUSTER_TO_JAWS_SITE = {
     # If we do that the perlmutter-jaws name is unfortunate since there are multiple virtual
     # sites there
     sites.Cluster.PERLMUTTER_JAWS: "kbase",  # what would they call a site actually at kbase...?
-    sites.Cluster.LAWRENCIUM_JAWS: "jgi",  # historical name
+    # sites.Cluster.LAWRENCIUM_JAWS: "jgi",  # historical name
 }
 
 
diff --git a/cdmtaskservice/jobflows/jaws_flows_provider.py b/cdmtaskservice/jobflows/jaws_flows_provider.py
index 4671be7..e6c6ae7 100644
--- a/cdmtaskservice/jobflows/jaws_flows_provider.py
+++ b/cdmtaskservice/jobflows/jaws_flows_provider.py
@@ -326,6 +326,6 @@ class JAWSFlowProvider:
             self._kafka,
             self._coman,
             self._service_root_url,
-            on_refdata_complete=lrcjawsflow.nersc_refdata_complete,
+            #on_refdata_complete=lrcjawsflow.nersc_refdata_complete,
         )
         return nerscjawsflow, lrcjawsflow
diff --git a/cdmtaskservice/jobflows/lawrencium_jaws.py b/cdmtaskservice/jobflows/lawrencium_jaws.py
index d4a004b..d29149f 100644
--- a/cdmtaskservice/jobflows/lawrencium_jaws.py
+++ b/cdmtaskservice/jobflows/lawrencium_jaws.py
@@ -16,7 +16,7 @@ from cdmtaskservice.notifications.kafka_notifications import KafkaNotifier
 from cdmtaskservice.mongo import MongoDAO
 from cdmtaskservice.nersc.manager import NERSCManager
 from cdmtaskservice.s3.client import S3ObjectMeta
-from cdmtaskservice import sites
+#from cdmtaskservice import sites
 from cdmtaskservice.update_state import submitted_refdata_download, refdata_complete
 
 
@@ -25,7 +25,7 @@ class LawrenciumJAWSRunner(NERSCJAWSRunner):
     Runs jobs at Lawrencium using JAWS.
     """
     
-    CLUSTER = sites.Cluster.LAWRENCIUM_JAWS
+    #CLUSTER = sites.Cluster.LAWRENCIUM_JAWS
     """ The cluster on which this runner operates. """
     
     def __init__(
diff --git a/cdmtaskservice/sites.py b/cdmtaskservice/sites.py
index 1cd8dc3..dffd085 100644
--- a/cdmtaskservice/sites.py
+++ b/cdmtaskservice/sites.py
@@ -7,17 +7,29 @@ from typing import Annotated
 from enum import Enum
 
 
+# The lawrencium site is currently not working with JAWS and plans to reenable it are
+# on hold indefinitely. Contact the JAWs program manager for more information.
+
+# Lines are currently commeneted out in 
+# * this file
+# * app_state.py
+# * jobflows/jaws_flows_provider.py
+# * jobflows/lawrencium_jaws.py
+# * jaws/sitemapper.py
+
+
 class Cluster(str, Enum):
     """
     The location where a job should run.
     
     perlmutter-jaws: The Perlmutter cluster at NERSC run via JAWS.
-    lawrencium-jaws: The Lawrencium cluster at LBNL run via JAWS.
     kbase: KBase compute nodes.
     """
+    
+    # lawrencium-jaws: The Lawrencium cluster at LBNL run via JAWS.
 
     PERLMUTTER_JAWS = "perlmutter-jaws"
-    LAWRENCIUM_JAWS = "lawrencium-jaws"
+    # LAWRENCIUM_JAWS = "lawrencium-jaws"
     KBASE = "kbase"
 
 
@@ -66,17 +78,17 @@ PERLMUTTER_JAWS = ComputeSite(
 
 
 # https://jaws-docs.jgi.doe.gov/en/latest/Resources/compute_resources.html
-LAWRENCIUM_JAWS = ComputeSite(
-    cluster=Cluster.LAWRENCIUM_JAWS,
-    nodes=8,
-    cpus_per_node=32,
-    memory_per_node_gb=492,  # in GB, not GiB, per the JAWS team
-    max_runtime_min=3 * 24 * 60 - 15,
-    notes=[
-        "The Lawrencium cluster at LBNL, serviced by the JAWS job running system.",
-        "Queue times are typically shorter here for smaller jobs."
-    ]
-)
+# LAWRENCIUM_JAWS = ComputeSite(
+#     cluster=Cluster.LAWRENCIUM_JAWS,
+#     nodes=8,
+#     cpus_per_node=32,
+#     memory_per_node_gb=492,  # in GB, not GiB, per the JAWS team
+#     max_runtime_min=3 * 24 * 60 - 15,
+#     notes=[
+#         "The Lawrencium cluster at LBNL, serviced by the JAWS job running system.",
+#         "Queue times are typically shorter here for smaller jobs."
+#     ]
+# )
 
 
 KBASE = ComputeSite(
@@ -96,7 +108,7 @@ KBASE = ComputeSite(
 
 CLUSTER_TO_SITE = {
     Cluster.PERLMUTTER_JAWS: PERLMUTTER_JAWS,
-    Cluster.LAWRENCIUM_JAWS: LAWRENCIUM_JAWS,
+    # Cluster.LAWRENCIUM_JAWS: LAWRENCIUM_JAWS,
     Cluster.KBASE: KBASE,
 }
 """ A mapping of compute clusters to their site information. """
@@ -104,7 +116,7 @@ CLUSTER_TO_SITE = {
 
 CLUSTER_TO_EXECUTION_TYPE = {
     Cluster.PERLMUTTER_JAWS: False,
-    Cluster.LAWRENCIUM_JAWS: False,
+    # Cluster.LAWRENCIUM_JAWS: False,
     Cluster.KBASE: True,
 }
 """

```

**Comment by MrCreosote** [comment]
Fixed in #608
